imaginedsf-custom-theme/acf-json/group_5914a7592f962.json

```diff
@@ -3,199 +3,42 @@
     "title": "Maps",
```

Comment:
I think you're missing a field for the linked basemap in the Maps field group.

Reply:
I have a field for linked basemap (starting line 44), but the type is 'Post Object' right now -- maybe that's what's wrong? Should I change it to text and give instructions to insert the link to the actual basemap?